src/test/ui/consts/const-address-of-mut.rs

```diff
@@ -1,14 +1,14 @@
 #![feature(raw_ref_op)]
 
-const A: () = { let mut x = 2; &raw mut x; };           //~ ERROR `&raw mut` is not allowed
+const A: () = { let mut x = 2; &raw mut x; };           //~ mutable reference
```

Comment:
Should this test's annotations still have `ERROR` in them?

```suggestion
const A: () = { let mut x = 2; &raw mut x; };           //~ ERROR mutable reference
```

Reply:
Why?

Reply:
I was under the impression that it was required for the annotation.

Reply:
it's good to have but not a necessity iirc.